software/fosscord-server/README.md

```diff
@@ -0,0 +1,5 @@
+# Fosscord-Server
+
+### From the [Fosscord-server](https://github.com/fosscord/fosscord-server) GitHub
+
+Run [Selfhosted discord](dev.fosscord.com), its selfhosted version of discord thats not connected to discord.com it has api,cdn and all that good stuff, for more information, checkout docs
```

Comment:
Using the official description of the project would be more fitting than "all that good stuff"

Reply:
> Using the official description of the project would be more fitting than "all that good stuff"

I'll do that rn